distributions/replication-dominion-pax-exam/pom.xml

```diff
@@ -35,13 +35,13 @@
         <dependency>
             <groupId>ddf.lib</groupId>
             <artifactId>dominion-commons</artifactId>
-            <version>${ddf.version}</version>
+            <version>${dominion.version}</version>
```

Comment:
❓ Is this because the dominion libraries have not been ported to the DDF branch we are now referencing? If so then this might actually cause a problem and these libraries should be ported over. Travis was working on that so we might want to check with him.

The fact is that the artifact version you are referencing is designed for the release branch and not the master branch of DDF which will cause issues or even not start properly DDF in some cases.

Reply:
Commented out all modules that use dominion and changed the versions back to just ddf.version